packages/cubejs-athena-driver/src/AthenaDriver.ts

```diff
@@ -335,7 +335,8 @@ export class AthenaDriver extends BaseDriver implements DriverInterface {
         const fields: Record<string, any> = {};
         columnInfo
           .forEach((c, j) => {
-            fields[c.Name] = row.Data?.[j].VarCharValue;
+            const _r = row.Data;
```

Comment:
Could you please replace `_r` with `r`? Names beginning with an underscore are unused fields according to our listing rules.